docs/docs/20-quickstart.md

```diff
@@ -568,269 +546,42 @@ the previous section.
     version of the `public.ecr.aws/nginx/nginx` container image.
     :::
 
-1. We'll use it later, so save the ID of the `Freight` to an environment
-   variable:
-
-    ```shell
-    export FREIGHT_ALIAS=$(kargo get freight --project kargo-demo --output jsonpath={.alias})
-    ```
-
-1. Now, let's _promote_ the `Freight` into the `test` `Stage`:
-
-    ```shell
-    kargo promote --project kargo-demo --freight-alias $FREIGHT_ALIAS --stage test
-    ```
-
-    Sample output:
-
-    ```shell
-    promotion.kargo.akuity.io/test.01j2wbtrym4r3tktv38qzteh5h.7a6e91f promotion created
-    ```
-
-    Query for `Promotion` resources within our project to see one has been
-    created:
+### Promote to Test
 
-    ```shell
-    kargo get promotions --project kargo-demo
-    ```
+1. To promote `Freight` to the `test` `Stage`, select the existing `Freight` from
+    the Freight Timeline above. Click the three dots, choose <Hlt>Promote</Hlt>,
+    and then select <Hlt>Promote</Hlt> that appeared on the `test` `Stage`.
+ 
+    The promotion may briefly appear in a `Pending` state, but it will likely
+    transition to `Running`, or even `Succeeded` within moments, as shown below:
```

Comment:
This wording probably made more sense with the CLI where you might need to issue a command to check on the status of the promotion a few times. The UI auto-refreshes and this is now more of a "blink and you may miss it" situation.

I would reword this, perhaps by taking some details you currently have farther down and moving them up here. Call out that when complete, there will be a checkmark. Call out that _after_ the Promotion is complete, a healthcheck will run periodically and when the Stage reaches a healthy state, it will be reflected with a heart.